### PR TITLE
feat: Notification page

### PR DESCRIPTION
**Pull request by Wennio-Oliveira**
# Description

Create Notification page.
Scenery: There is no new notifications

[Jira Ticket](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-545)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manually tested.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-webapp-69-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar-webapp/actions/workflows/merge-main.yml)

### CODE REVIEW

File: frontend/src/components/NotificationsCentral/index.tsx [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+  Row,
+} from '@carbon/react';
+import { Notification } from '@carbon/icons-react';
 
 const NotificationsCentral = () => (
-  <span> Placeholder </span>
+  <div className="notifications-space">
+    <Row>
+      <Button className='btn-viewed' kind="ghost">Mark all as viewed</Button>
+      <Button className='btn-all-notification' kind="ghost">See all notification</Button>
```
Comment (RMCampos) [inline frontend/src/components/NotificationsCentral/index.tsx:14]: `notification` should be in the plural form.

File: frontend/src/components/NotificationsCentral/index.tsx [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
+  <div className="notifications-space">
+    <Row>
+      <Button className='btn-viewed' kind="ghost">Mark all as viewed</Button>
+      <Button className='btn-all-notification' kind="ghost">See all notification</Button>  
+    </Row>
+    <hr></hr>
+    <div className="container">
+      <div className="vertical-center">
+        <EmptySection
+          icon="Sun"
```
Comment (RMCampos) [inline frontend/src/components/NotificationsCentral/index.tsx:20]: Another icon should be used here.

File: frontend/src/components/NotificationsCentral/index.tsx [hunk 3]
```diff
@@ -1,7 +1,35 @@
...
+          title="There is no new notifications!"
+          description="You can go to all notifications to see older ones."
```
Comment (RMCampos) [inline frontend/src/components/NotificationsCentral/index.tsx:22]: The description is somehow bolder. On Figma this piece of text is not that bold.

**Review by RMCampos — CHANGES_REQUESTED** [review]
Great job!! There are tests failing, and some fixes to address, though. ✅

File: frontend/src/components/NotificationsCentral/index.tsx (hunk 2 above)
Comment (Wennio-Oliveira) [reply frontend/src/components/NotificationsCentral/index.tsx:20]: After Craig's PR to be merged I'll back to fix it.

File: frontend/src/components/NotificationsCentral/index.tsx [hunk 4]
```diff
@@ -11,7 +11,7 @@ const NotificationsCentral = () => (
   <div className="notifications-space">
     <Row>
       <Button className='btn-viewed' kind="ghost">Mark all as viewed</Button>
-      <Button className='btn-all-notification' kind="ghost">See all notification</Button>  
+      <Button className='btn-all-notifications' kind="ghost">See all notification</Button>
```
Comment (RMCampos) [inline frontend/src/components/NotificationsCentral/index.tsx:14]: Still missing to fix `See all notification` to plural form 😆

**Review by RMCampos — CHANGES_REQUESTED** [review]

File: frontend/src/components/NotificationsCentral/style.scss [hunk 5]
```diff
@@ -0,0 +1,42 @@
+@use '../../theme/variables.scss' as vars;
+
+.#{vars.$bcgov-prefix}--row{
```
Comment (mgaseta) [inline frontend/src/components/NotificationsCentral/style.scss:3]: Add a custom class to the Row component tag, something like `className="notifications-button-row"`, and use that class name to apply styles. Using directly `.#{vars.$bcgov-prefix}--row` will make the changes globally to all rows.

File: frontend/src/components/NotificationsCentral/style.scss [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  padding-right: 3.31rem;
+}
+
+.empty-section-container {
+  padding: 1.5rem 1rem;
+}
+
+p.subtitle-section {
+  font-size: 0.875rem;
+  font-weight: normal;
```
Comment (mgaseta) [inline frontend/src/components/NotificationsCentral/style.scss:21]: Use carbon types here, so it remains on the same pattern, take a look at the PageTitle's `style.scss` (lines 1 and 8). The type style can be seen on figma.

File: frontend/src/components/NotificationsCentral/style.scss [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+.btn-viewed {
+  padding-right: 3.75rem;
+}
+
+.btn-all-notifications {
+  padding-right: 3.31rem;
+}
+
+.empty-section-container {
```
Comment (mgaseta) [inline frontend/src/components/NotificationsCentral/style.scss:15]: Use other class exclusively of your changes to make this css rule more specific, e.g.: `vertical-center .empty-section-container`.

File: frontend/src/components/NotificationsCentral/index.tsx [hunk 8]
```diff
@@ -1,7 +1,35 @@
...
 const NotificationsCentral = () => (
-  <span> Placeholder </span>
+  <div className="notifications-space">
+    <Row>
+      <Button className='btn-viewed' kind="ghost">Mark all as viewed</Button>
+      <Button className='btn-all-notifications' kind="ghost">See all notifications</Button>  
+    </Row>
+    <hr></hr>
+    <div className="container">
+      <div className="vertical-center">
```
Comment (mgaseta) [inline frontend/src/components/NotificationsCentral/index.tsx:18]: These classes names are too generic, make them more specific to this notifications central section, like `notifications-central-container`

File: frontend/src/components/NotificationsCentral/style.scss [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+  padding: 1.5rem 1rem;
+}
+
+p.subtitle-section {
+  font-size: 0.875rem;
+  font-weight: normal;
+}
+
+hr {
+  border-top: 1px solid #DFDFE1;
```
Comment (mgaseta) [inline frontend/src/components/NotificationsCentral/style.scss:25]: Use a defined css var to apply the color, I think `--bcgov-border-subtle-00` is the correct one for this case, but please confirm on figma.

**Review by mgaseta — CHANGES_REQUESTED** [review]
Very good job 🚀🎉, just added a few markups.

**Review by lucasbuenoencora — COMMENTED** [review]
LGTM! Needs to update with the latest changes from main

**Review by mgaseta — APPROVED** [review]
LGTM ✔🥇🎉!!

**Pull request merged**